stripe-core/src/main/java/com/stripe/android/core/exception/LocalStripeException.kt

```diff
@@ -0,0 +1,10 @@
+package com.stripe.android.core.exception
+
+import androidx.annotation.RestrictTo
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+class LocalStripeException(
+    message: String?
```

Comment:
Should we make this more clear that this is a displayMessage?

Reply:
Yeah, good idea. Thanks!